### DISCUSSION TITLE
Беда

### DISCUSSION THREAD

**Discussion by evgeniy-152** [Chia-Network/chia-blockchain#8425] [Support]
Здравствуйте. После обновления пропали участки и не могут обратно добавиться. Могу добавить только один любой диск с участками а остальные не получается. После удаления всех директорий с участками ничего не изменилось, приложение может добавить только одну директорию с участками.

**Comment by loppefaaret** [comment]
could you perhaps try and open up `debug.log` in your users folder `~\.chia\mainnet\log\` and see if there are any ERROR reported ?

**Comment by BrandtH22** [comment]
@evgeniy-152 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
